### DISCUSSION TITLE
Why is the monitoring metric milvus_proxy_sq_latency not found in Milvus's metrics?

### DISCUSSION THREAD

**Discussion by ahern88** [milvus-io/milvus#35484] [Q&A and General discussion] [answered]
@xiaofan-luan

Below is the monitoring metrics I observed through Prometheus, and I did not find milvus_proxy_sq_latency mentioned in the document.

please see： https://milvus.io/docs/metrics_dashboard.md

`
Search Latency | The average latency and the 99th percentile of the latency of receiving search and query requests by each proxy within the past two minutes. | p99:histogram_quantile(0.99, sum by (le, query_type, pod, node_id) (rate(milvus_proxy_sq_latency_bucket{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])))avg:sum(increase(milvus_proxy_sq_latency_sum{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])) by (pod, node_id, query_type) / sum(increase(milvus_proxy_sq_latency_count{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])) by (pod, node_id, query_type) | milvus_proxy_sq_latency | The latency of search and query requests.
-- | -- | -- | -- | --

`

`milvus_build_info

milvus_datacoord_channel_checkpoint_unix_seconds

milvus_datacoord_collection_num

milvus_datacoord_consume_datanode_tt_lag_ms

milvus_datacoord_datanode_num

milvus_datacoord_import_tasks

milvus_datacoord_index_node_num

milvus_datacoord_segment_num

milvus_datacoord_stored_binlog_size

milvus_datacoord_stored_index_files_size

milvus_datacoord_stored_rows_num

milvus_datacoord_watched_dml_chanel_num

milvus_flushed_segment_file_num_bucket

milvus_flushed_segment_file_num_count

milvus_flushed_segment_file_num_sum

milvus_meta_kv_size_bucket

milvus_meta_kv_size_count

milvus_meta_kv_size_sum

milvus_meta_op_count

milvus_meta_request_latency_bucket

milvus_meta_request_latency_count

milvus_meta_request_latency_sum

milvus_msgstream_op_count

milvus_msgstream_request_latency_bucket

milvus_msgstream_request_latency_count

milvus_msgstream_request_latency_sum

milvus_num_node

milvus_proxy_apply_timestamp_latency_bucket

milvus_proxy_apply_timestamp_latency_count

milvus_proxy_apply_timestamp_latency_sum

milvus_proxy_cache_hit_count

milvus_proxy_cache_update_latency_bucket

milvus_proxy_cache_update_latency_count

milvus_proxy_cache_update_latency_sum

milvus_proxy_collection_sq_latency_bucket

milvus_proxy_collection_sq_latency_count

milvus_proxy_collection_sq_latency_sum

milvus_proxy_executing_total_nq

milvus_proxy_rate_limit_req_count

milvus_proxy_receive_bytes_count

milvus_proxy_received_nq

milvus_proxy_report_value

milvus_proxy_req_count

milvus_proxy_search_vectors_count

milvus_proxy_send_bytes_count

milvus_proxy_sq_latency_bucket

milvus_proxy_sq_latency_count

milvus_proxy_sq_latency_sum

milvus_proxy_sq_reduce_result_latency_bucket

milvus_proxy_sq_reduce_result_latency_count

milvus_proxy_sq_reduce_result_latency_sum

milvus_proxy_sq_wait_result_latency_bucket

milvus_proxy_sq_wait_result_latency_count

milvus_proxy_sq_wait_result_latency_sum

milvus_proxy_tt_lag_ms

milvus_proxy_workload_score

milvus_querycoord_collection_num

milvus_querycoord_current_target_checkpoint_unix_seconds

milvus_querycoord_load_latency_bucket

milvus_querycoord_load_latency_count

milvus_querycoord_load_latency_sum

milvus_querycoord_partition_num

milvus_querycoord_querynode_num

milvus_querycoord_task_latency_bucket

milvus_querycoord_task_latency_count

milvus_querycoord_task_latency_sum

milvus_querycoord_task_num

milvus_rootcoord_collection_num

milvus_rootcoord_credential_num

milvus_rootcoord_ddl_req_count

milvus_rootcoord_ddl_req_latency_bucket

milvus_rootcoord_ddl_req_latency_count

milvus_rootcoord_ddl_req_latency_in_queue_bucket

milvus_rootcoord_ddl_req_latency_in_queue_count

milvus_rootcoord_ddl_req_latency_in_queue_sum

milvus_rootcoord_ddl_req_latency_sum

milvus_rootcoord_dml_channel_num

milvus_rootcoord_entity_num

milvus_rootcoord_id_alloc_count

milvus_rootcoord_indexed_entity_num

milvus_rootcoord_msgstream_obj_num

milvus_rootcoord_num_of_roles

milvus_rootcoord_partition_num

milvus_rootcoord_produce_tt_lag_ms

milvus_rootcoord_proxy_num

milvus_rootcoord_sync_timetick_latency_bucket

milvus_rootcoord_sync_timetick_latency_count

milvus_rootcoord_sync_timetick_latency_sum

milvus_rootcoord_timestamp

milvus_rootcoord_timestamp_saved

milvus_runtime_info`

**Comment by LoveEachDay** [comment, marked as answer]
@ahern88 The following three metrics: 
- milvus_proxy_sq_latency_bucket
- milvus_proxy_sq_latency_count
- milvus_proxy_sq_latency_sum

are all for the histogram metrics `milvus_proxy_sq_latency`.

**Reply by ahern88** [reply]
I understand, thank you for your reply.
